From 0633abfadaf9a2652dd974fb113e504c836d9d8c Mon Sep 17 00:00:00 2001
From: cnbuild <cnbuild>
Date: Fri, 31 May 2024 03:20:54 +0000
Subject: [PATCH] Add cp0_rtc and cp1_rtc node definitions from
 0020-arm64-dts-cn9131-cf-solidwan-enable-only-cp0-rtc.patch

---
 arch/arm64/boot/dts/marvell/cn9130-cf-solidwan.dts | 4 ++++
 arch/arm64/boot/dts/marvell/cn9131-cf-solidwan.dts | 5 +++++
 2 files changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/cn9130-cf-solidwan.dts b/arch/arm64/boot/dts/marvell/cn9130-cf-solidwan.dts
index 02d8e66422dd..479d5de9c463 100644
--- a/arch/arm64/boot/dts/marvell/cn9130-cf-solidwan.dts
+++ b/arch/arm64/boot/dts/marvell/cn9130-cf-solidwan.dts
@@ -411,3 +411,7 @@ cp0_uart1_pins: cp0-uart1-pins {
 
 	};
 };
+
+&cp0_rtc {
+       status = "okay";
+};
diff --git a/arch/arm64/boot/dts/marvell/cn9131-cf-solidwan.dts b/arch/arm64/boot/dts/marvell/cn9131-cf-solidwan.dts
index f5af014da8d5..6d0e433c3816 100644
--- a/arch/arm64/boot/dts/marvell/cn9131-cf-solidwan.dts
+++ b/arch/arm64/boot/dts/marvell/cn9131-cf-solidwan.dts
@@ -238,3 +238,8 @@ cp1_uart1_pins: cp1-uart1-pins-1 {
 	};
 };
 
+
+&cp1_rtc {
+       status = "disabled";
+};
+
-- 
2.43.0

